pkg/executor/backoff_test.go

```diff
@@ -45,42 +45,53 @@ var _ = Describe("exponential limited backoff", func() {
 		testsClock.Step(time.Nanosecond)
 	})
 
-	It("should be ready before first step", func() {
+	It("should be ready and not timeout before first step", func() {
```

Comment:
I do not think any changes are needed in the description, because the scenario is not saying anything about the max time passing. But if you insist, perhaps make it something like:
> should be ready before first step and before max time reached

Same for the others.
But my suggestion is not to add anything. Up to you.

Reply:
yeah I'm convinced it's enough